src/main/java/spoon/reflect/declaration/CtType.java

```diff
@@ -234,6 +234,14 @@
 	@PropertyGetter(role = METHOD)
 	<R> CtMethod<R> getMethod(String name, CtTypeReference<?>... parameterTypes);
 
+	/**
+	 * Gets a method from its signature.
+	 *
+	 * @return null if does not exit
+	 */
+	@PropertyGetter(role = METHOD)
+	<R> CtMethod<R> getMethodBySignature(String signature);
```

Comment:
I don't quite see how this relates to the title and scope of this PR. This should be done in a separate PR IMHO.

Reply:
Same to constructor, a CtMethod has a CtPath ends with #method[signature=(...)], I need this function to get the CtElement.